source/_posts/2021-04-07-release-20214.markdown

```diff
@@ -56,28 +77,130 @@ Screenshot of a lorem ipsum occurrence in Home Assistant.
 
 ## Home Assistant Analytics
 
-Lorem ipsum
+Now don't be scared by the title!
 
-https://analytics.home-assistant.io
+Today we introduce, Home Assistant Analytics. Opt-in, privacy aware, public
+and open source. Just as it all should be for our project.
 
-## Filtering automations
+Ludeeus did an amazing job writing this new integration (that replaces the
+updater). You decide if you turn it on or not; however, we guarantee its privacy
```

Comment:
```suggestion
Ludeeus did an amazing job writing this new integration (that replaces the
analytics part of the updater). You decide if you turn it on or not; however, we guarantee its privacy
```
?
The other function of that integration is still intact

Reply:
I read it as the new integration replaces the old completely, good enough for now :D